mailbox/api/src/main/java/org/apache/james/mailbox/acl/SimpleGroupMembershipResolver.java

```diff
@@ -68,7 +68,7 @@ public String toString() {
 
     }
 
-    private final Set<Membership> memberships = new HashSet<SimpleGroupMembershipResolver.Membership>(32);
+    private final Set<Membership> memberships = new HashSet<>(32);
```

Comment:
Let's eclipse guys correct this later on where need be?

It is not in my IDE, and no ERRROR raised in **maven**. However Eclipse guys raise a lot of errors in my IDE :trollface:  .